sql/sql_class.h

```diff
@@ -2735,7 +2737,7 @@ class THD: public THD_count, /* this must be first */
   */
   mutable mysql_mutex_t LOCK_thd_kill;
 
-  open_address_hash<table_key, hash_trait<TABLE_LIST, TABLE_LIST*>> pr_table_hash;
+  open_address_hash<MDL_key_trait<TABLE_LIST>, hash_trait<TABLE_LIST, TABLE_LIST *>> pr_table_hash;
```

Comment:
better make a space after the first closing `>` bracket -- not all C++ parsers are good at distinguishing it with a shift operator, and though it's now covered by 11 standard, still can be a good gesture